hudi-common/src/main/java/org/apache/hudi/common/model/HoodieCommitMetadata.java

```diff
@@ -236,6 +240,40 @@ public static <T> T fromJsonString(String jsonStr, Class<T> clazz) throws Except
     return JsonUtils.getObjectMapper().readValue(jsonStr, clazz);
   }
 
+  /**
+   * parse the bytes of deltacommit, and get the base file and the log files belonging to this
+   * provided file group.
+   */
+  public static Pair<String, List<String>> getFileSliceForFileGroupFromDeltaCommit(
```

Comment:
does nt any of the existing code do this?

Reply:
no. let me do this first. no simple and low-codes way can do this. i think it deserves a new pr.